src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] NUMBERS = new int[12];
+	private static final int INDEX_OF_EMPTY_STACK = -1;
+    private static final int STACK_CAPACITY = 12;
+    private static final int INDEX_OF_FULL_STACK = 11;
 
-    public int total = -1;
+    private final int[] numbers = new int[STACK_CAPACITY];
+
+    private int total = INDEX_OF_EMPTY_STACK;
 
     public void countIn(int in) {
         if (!isFull())
-            NUMBERS[++total] = in;
+            numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == INDEX_OF_EMPTY_STACK;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == INDEX_OF_FULL_STACK;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
-        return NUMBERS[total];
+            return INDEX_OF_EMPTY_STACK;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)